torch/onnx/_internal/diagnostics/rules.yaml

```diff
@@ -83,178 +79,98 @@
     deprecated: false
     tags: []
 
+## Rules for PyTorch (dynamo fx based) ONNX Exporter (FXE)
 
-
-- id: FXE0001
-  name: fx-tracer-success
+- id: FXE0007
```

Comment:
Should we re-start from one?

Reply:
I'm leaving as is, technically it is taken and exposed in 2.0 ..